app/views/organizations/index.html.erb

```diff
@@ -1,41 +1,51 @@
-<div>
-  <div id="map" style='height: 280px;'></div>
-</div>
+<div class = "header-container" >
```

Comment:
Looking at this file, it's really hard to find where this `div` closes. Does it close? Or, maybe your indenting should be fixed.